SWIFTLINT.md

```diff
@@ -0,0 +1,138 @@
+* [Approved exceptions](#approved-exceptions)
+	* [Implicitly Unwrapped Optionals](#implicitly-unwrapped-optionals)
+	* [Force cast](#force-cast)
+	* [Force unwrapping](#force-unwrapping)
+	* [SwiftUI and multiple trailing closures](#swiftui-and-multiple-trailing-closures)
+	* [Open-source files](#open-source-files)
+* [Other notes](#other-notes)
+
+## Installing SwiftLint
+
+
+```bash
+brew install swiftlint
+```
+
+If you are unable to use Homebrew, you may use one of the other methods described in the [SwiftLint documentation](https://github.com/realm/SwiftLint).
+
+**Do not** install SwiftLint as a CocoaPod in your project.
+
+
+## Xcode settings
+
+You'll need to configure Xcode to remove trailing whitespace from all lines. This is not the default configuration.
+
+In Xcode's Preferences, select **Text Editing** ▸ **Editing** and check **Including whitespace-only lines**. 
+
+![](screens/trailing-whitespace.png)
+
+## Handling rule exceptions
+
+Your sample project must compile without warnings — SwiftLint or otherwise. In general, you should change your code to eliminate all warnings where necessary. When it comes to SwiftLint, however, there will be times when this isn't possible. In these situations, you'll need to use in-line comments to temporarily disable rules. You can find appropriate syntax to do this in [the SwiftLint documentation](https://realm.github.io/SwiftLint/#disable-rules-in-code).
+
+You may only disable a rule if it is on the list of approved exceptions listed below.
+
+Prefer the form that disables a rule only for the next line:
+```
+// swiftlint:disable:next implicitly_unwrapped_optional
+var injectedValue: Data!
+```
+
+Or, similarly, for the previous line:
+```
+var injectedValue: Data!
+// swiftlint:disable:previous implicitly_unwrapped_optional
+```
+
+If there are several approved exceptions, group them together and disable the rule for that region. Be sure to enable the rule after that section. Do not leave a rule disabled throughout the source file.
+
+```
+// swiftlint:disable implicitly_unwrapped_optional
+var injectedValue1: Data!
+var injectedValue2: Data!
+// swiftlint:enable implicitly_unwrapped_optional
+```
+
+If you must disable rules in your project, leave those in-line comments in the project for the benefit of your teammates in the editing pipeline.
+
+Finally, if you're not sure which rule is triggering a warning, you can find the rule name in parentheses at the end of message:
+
+![](screens/swiftlint-warning.png)
+
+## Approved exceptions
+
+There are certain common idioms that violate SwiftLint's strict checking. If you are unable to work around them — and you've already tried to find a better way to structure your code — you may disable rules as described in this section.
+
+If you find that you're struggling with rules other than those described below, please reach out to your Team Lead with your specific example.
+
+### Implicitly Unwrapped Optionals
```

Comment:
Minor text consistency: this is the only header that's using capitals for each word.